src/remove.js

```diff
@@ -0,0 +1,13 @@
+const remove = (array, fn) => {
+if(array === null || array === undefined){ return undefined}
```

Comment:
Indentation is off.  For clarity, I recommend putting each statement on its own line:
```js
if( array === null || array === undefined ) {
  return undefined
}
```